webhook/webhook.py

```diff
@@ -1,23 +1,27 @@
-import sys
-import time
-import json
 import time
 from flask import Flask, request
 import os, subprocess
+import tempfile
+import shutil
 from onepasswordconnectsdk.client import Client, new_client
 import onepasswordconnectsdk
 
 ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
 ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
-VAULT_ID = "quvhrzaatbj2wotsjrumx3f62a"
-SECRET_NAME = "airflow.fyi webhook key"
+VAULT_ID = os.getenv("OP_VAULT_ID")
+OPERATING_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
```

Comment:
While we have "leaked" our vault ID, it's not something we need to be concerned about. It's secured, and there isn't any changing it.